### ISSUE TITLE
DOC: Example frontend tests

### ISSUE THREAD

**Issue by matthewturk** [yt-project/yt#4111]
We should have a checklist of frontend tests that are required for inclusion of a new frontend.

**Comment by chummels** [comment]
Good idea.  Perhaps listing this in the docs as well as a framework/note in the skeleton frontend.
